src/main/java/me/botsko/prism/utils/block/Utilities.java

```diff
@@ -774,10 +650,18 @@ public static boolean areBlockIdsSameCoreItem(Material mat1, Material mat2) {
         if (mat1 == mat2) {
             return true;
         }
+        mat1 = getMaterial(mat1);
+        mat2 = getMaterial(mat2);
 
-        mat1 = baseMaterials.get(mat1);
-        mat2 = baseMaterials.get(mat2);
+        return  mat1 == mat2;
```

Comment:
Codacy found an issue: [Use equals() to compare object references.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5612535)